data/schema-config.yaml

```diff
@@ -436,28 +436,44 @@ translates to:
     - RO:0002513
     - SIO:010082
 
-asb:
+allele specific binding: ## Need to add acc, filter
```

Comment:
I'm working on the endpoints, I can add them for you later. We can remove the comment. A few yaml file parsers might not support the ## tags